### ISSUE TITLE
After expanding the Combo box and swipe narrator focus is not going to combo list items.

### ISSUE THREAD

**Issue by anjais123** [microsoft/fluentui#15064]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/khali007/pen/yLYvOVR?editors=1010

### Repro Steps:
Scree Reader + Narrator
1. Turn on narrator​
2. Swipe till Combo box and double tap to expand​
3. Swipe and verify narrator focus is going to the list items or not​.

#### Actual behavior:
After expanding the combo box and swipe narrator focus is not going to combo list items​.

#### Expected behavior:
After expanding the activity type combo box when user swipe,  narrator focus should go to combo list items​. User should be able to select different comboBox list items using swipe.

### User Impact:​
Visually impaired users will not be able to access those activity type combo list items when narrator is ON​.

**Comment by paulgildea** [comment]
@anjais123 Thanks for the feedback.
